docs/quest_template.md

```diff
@@ -223,3 +223,399 @@ Contains all basic definitions of available quests.
+| 1     | Quest is disabled (not yet implemented in the core).                                                     |
+| 2     | Quest is enabled (does not auto-complete).                                                               |
+| 3     | Quest is a World Quest.                                                                                  |
+
+### QuestLevel
+
+Level of quest. Player receives full experience amount only if their level is less than or equal to Level+5. If Level is set to -1, the player's level will be used as (Quest)Level for the experience calculation.
+
+### MinLevel
+
+Minimum level at which a player can get the quest.
+
+### QuestSortID
+
+This field defines under what category the quest falls in the quest log.
+
+If **value &gt; 0** then value is Zone IDs taken from AreaTable.dbc.
+
+if **value &lt; 0** then (**-value**) is quest sort id: (in general profession or class quests. Also see [RequiredSkillPoints](#quest_template-RequiredSkillPoints) ) Value is ID from QuestSort.dbc
+
+### QuestInfoID
+
+These values are ID taken from [QuestInfo.dbc](QuestInfo)
+
+### SuggestedGroupNum
+
+Recommended number of players to do the quest together.
+
+### RequiredFactionId1
+
+Faction ID 1 for an objective to achieve a certain reputation value with. See Faction.dbc
+
+### RequiredFactionId2
+
+Faction ID 2 for an objective to achieve a certain reputation value with. See Faction.dbc
+
+### RequiredFactionValue1
+
+Reputation value 1 that the player must achieve with the faction in [RepObjectiveFaction](#quest_template-RepObjectiveFaction) as part of the quest objectives.
+
+### RequiredFactionValue2
+
+Reputation value 2 that the player must achieve with the faction in [RepObjectiveFaction](#quest_template-RepObjectiveFaction) as part of the quest objectives.
+
+### RewardNextQuest
+
+**(Old field name: NextQuestIdChain)**
+
+The quest entry from a **creature** or **gameobject** that ends a quest and starts a new one. The result is, that if you end the quest, the new quest instantly appears from the quest giver.
+
+See the [examples section](#quest_template-Examples) for examples.
+
+### RewardXPDifficulty
+
+According to the [Level](#quest_template-Level), the basic experience with index *RewardXPDifficulty* is taken from QuestXP.dbc.
+
+This field also controls the XP given as the XP is calculated from the value in this field by the following formula. If the quest is repeatable, XP will be given only once. The total XP that a character will receive is also affected by the level difference between the character's level and the quest's level.
+
+The formula for calculating XP from the value in this field:
+**QuestLevel &gt;= 65:** XP = RewMoneyMaxLevel / 6.0
+**QuestLevel h1. 64:** XP = RewMoneyMaxLevel / 4.8
+**QuestLevel 63:** XP = RewMoneyMaxLevel / 3.6
+**QuestLevel h1. 62:** XP = RewMoneyMaxLevel / 2.4
+**QuestLevel 61:** XP = RewMoneyMaxLevel / 1.2
+**QuestLevel &lt;= 60:** XP = RewMoneyMaxLevel / 0.6
```

Comment:
You need a blankline between to show on different lines.

Now this will show like this
```
The formula for calculating XP from the value in this field:**QuestLevel &gt;= 65:** XP = RewMoneyMaxLevel / 6.0**QuestLevel h1. 64:** XP = RewMoneyMaxLevel / 4.8**QuestLevel 63:** XP = RewMoneyMaxLevel / 3.6**QuestLevel h1. 62:** XP = RewMoneyMaxLevel / 2.4**QuestLevel 61:** XP = RewMoneyMaxLevel / 1.2**QuestLevel &lt;= 60:** XP = RewMoneyMaxLevel / 0.6

Reply:
Corrected.